platform/ethereum/domain.go

```diff
@@ -32,10 +51,10 @@ func (p *Platform) Lookup(coins []uint64, name string) ([]blockatlas.Resolved, e
 	return result, nil
 }
 
-func (p *Platform) addressForCoin(resovler string, node []byte, coin uint64) (string, error) {
-	result, err := p.ens.Addr(resovler, node, coin)
+func (p *Platform) addressForCoin(resovler string, node []byte, cionInt uint64) (string, error) {
+	result, err := p.ens.Addr(resovler, node, cionInt)
 	if err != nil {
-		if coin == CoinType.ETH {
+		if cionInt == coin.ETH {
```

Comment:
cion?
cionInt of type uint?

Reply:
coin is taken now by the package name as alias is removed, cannot use coin

Reply:
not coin, but not cion and please do not put type into the variable name, it it weird that:
1. it is grammar mistake
2. it is called int but it is uint

Reply:
Point about type is good, typo is bad.  'coinType' it is now.